account/account.go

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package account
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/network"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/key"
+)
+
+type Account struct {
+	*key.SoftKey
```

Comment:
is Account always going to operate on the soft key or soft key will always be part of it? I just wonder if we should use Key interface here instead

Reply:
based on our previous discussion the interface we are having is wallet and accounts. 

accounts = keychain in avalanchego world